doc: known issues: Add KRKNWK-19263 related to Zigbee FOTA bug on nRF53 (#17813)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

KRKNWK-6073: Potential delay during FOTA
  There might be a noticeable delay (~220 ms) between calling the ZBOSS API and on-the-air activity.

> .. rst-class:: v2-7-0

Author: 👍 done

> KRKNWK-19263: FOTA DFU on nRF5340 fails due to invalid update image

Reviewer: ```suggestion
KRKNWK-19263: FOTA DFU on the nRF5340 DK fails due to an invalid update image
```

Author: 👍 fixed

>   Performing fota dfu on nrf5340 throws an error during an attempt to install the new image.

Reviewer: ```suggestion
  Performing FOTA DFU throws an error during an attempt to install the new image.
  
  **Affected platforms:** nRF5340
```

Author: fixed

>   **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``to be updated: pull/17796 not yet merged``).

Reviewer: ```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix from ``main`` (commit hash: ``to be updated: pull/17796 not yet merged``).
```

Author: fixed
